services.yaml

```diff
@@ -0,0 +1,10 @@
+- Athena
+- Batch
+- Comprehend Medical
```

Comment:
@chueatwork think we can drop this to just Comprehend? It'll be a very long tag otherwise.

Reply:
Yeah I was going to ask about character limit.

Did I really leave of SNS, wth

Reply:
No hard limit, but just figured it'd be really long. Thanks!